runtime/src/test/identity.rs

```diff
@@ -69,6 +71,76 @@ fn only_claim_issuers_can_add_claims() {
     });
 }
 
+#[test]
+fn only_claim_issuers_can_add_claims_batch() {
+    with_externalities(&mut build_ext(), || {
+        let owner_did = register_keyring_account(AccountKeyring::Alice).unwrap();
+        let issuer_did = register_keyring_account(AccountKeyring::Bob).unwrap();
+        let issuer = AccountKeyring::Bob.public();
+        let claim_issuer_did = register_keyring_account(AccountKeyring::Charlie).unwrap();
+        let claim_issuer = AccountKeyring::Charlie.public();
+        let mut claim_records = Vec::new();
+        let claim_key = "key".as_bytes();
+        claim_records.push(ClaimRecord::new(
```

Comment:
A better way to generate `claim_records` (functional languages would always rather inmutable over mutable) : 
```
let claim_records = vec![ ClaimRecord { ... }, ClaimRecord {...}];
```